gradle.properties

```diff
@@ -2,15 +2,14 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Fabric Properties
-	minecraft_version=1.16.5
-	yarn_mappings=1.16.5+build.6
+	minecraft_version=1.17-pre2
+	yarn_mappings=1.17-pre2+build.1
 	loader_version=0.11.3
 
 #Fabric api
-	fabric_version=0.32.5+1.16
+	fabric_version=0.34.8+1.17
 
 # Mod Properties
-	mod_version = 1.0.3
+	mod_version = 1.0.4
 	maven_group = net.consistencyteam
 	archives_base_name = obsidian_slab
-

```

Comment:
what happened here with the line breaks?

Reply:
I have no idea. I tried switching from CRLF to LF, but that changed nothing. Frankly, that made it worse, and it stayed like that after i removed the commit...